techniques/failures/F83.html

```diff
@@ -51,85 +49,84 @@
-   </section><section id="examples"><h2>Examples</h2>
+<section id="description"><h2>解説</h2>
+      <p>この失敗例は、ロービジョンの人が背景画像の上に表示されているテキストを読むことができない場合に関するものである。背景画像とテキストのコントラストが十分ではない時、背景画像はテキストと混同し、正確に読むことを困難にさせることがある。</p>
+      <p>達成基準 1.4.3 及び 1.4.6 を満たすためには、テキストとその背景画像の間に十分なコントラストを確保しなければならない。画像に対して、これはテキストの背後にあってテキストにとても似ている画像の一部とテキストの間に十分なコントラストが必要であることを意味している。</p>
+   </section><section id="examples"><h2>事例</h2>
       <section class="example" id="failure-example-1">
-         <h3>Failure Example 1</h3>
+         <h3>失敗例 1</h3>
          
-            <p>Black text overlays an image with black lines. The lines cross behind the letters making F's look like E's etc.</p>
+            <p>黒いテキストが黒い線の画像と重なっている。その線は文字の背景にあり、F's をまるで E's のように見せている。</p>
          
       </section>
       <section class="example" id="failure-example-2">
-         <h3>Failure Example 2</h3>
+         <h3>失敗例 2</h3>
          
-            <p>Black text overlays an image with dark gray areas. Wherever the text crosses a dark gray area the contrast is so bad that the text cannot be read.</p>
+            <p>黒いテキストが濃い灰色の場所のある画像と重なっている。テキストが濃い灰色のどこの場所で交差していても、コントラストがとても不十分なため、テキストを読むことができない。</p>
          
       </section>
    </section>
 
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
          <ol>
             <li> 
-                  <strong>Quickcheck:</strong> First do a quick check to see if the contrast between the text and the area of the image that is darkest (for dark text) or lightest (for light text) meets or exceeds that required by the Success Criterion (1.4.3 or 1.4.5). If the contrast meets or exceeds the specified contrast, then there is no failure. </li>
-            <li>If the Quickcheck is false, then check to see if the background behind each letter has sufficient contrast with the letter.</li>
+                  <strong>クイックチェック:</strong> まず初めに、テキストとその場所の一番暗い部分 (色の暗いテキストに対して) 又は一番明るい部分 (色の明るいテキストに対して) との間のコントラストが達成基準 (1.4.3 又は 1.4.5) による要求事項を満たしている、又は上回っていることを確認する。コントラストが指定されたコントラスト以上の場合、失敗しない。</li>
+            <li>クイックチェックが偽である場合、各文字の背後にある背景がその文字と十分なコントラストがあるかどうかを確認する。</li>
          </ol>
       </section>
-      <section class="results" id="expected-results"><h3>Expected Results</h3>
+      <section class="results" id="expected-results"><h3>期待される結果</h3>
          <ul>
-            <li>If Quickcheck is false and #2 is false as well then this failure condition applies and the content fails the contrast Success Criterion.</li>
+            <li>クイックチェックが偽かつ 2 も同様にが偽である場合、この失敗条件は適用され、コンテンツは達成基準に失敗する。</li>
```

Comment:
単純な TYPO だと思いますが、念のためご確認いただけますと幸いです。
```suggestion
            <li>クイックチェックが偽かつ 2 も同様に偽である場合、この失敗条件は適用され、コンテンツは達成基準に失敗する。</li>
```